### DISCUSSION TITLE
can I temporarily disable a mapping

### DISCUSSION THREAD

**Discussion by ghistes** [kovidgoyal/kitty#6503] [Q&A]
Hi,

I have "map ctrl+v paste_from_clipboard" in my config, but sometimes I want to run a program that also uses ctrl+v.

My question: Is there a way to temporarily disable such a mapping in kitty, so that I could disable it, run the program and re-enable it once I am finished?

Many thanks!

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/conf/#shortcut-kitty.Reload-kitty.conf
